### PR TITLE
mdxmini 2.0.0

### PR DESCRIPTION
**Pull request by mistydemeo**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/mdxmini.rb [hunk 1]
```diff
@@ -16,17 +16,15 @@ class Mdxmini < Formula
...
   end
 
   def install
-    # Specify Homebrew's cc
-    inreplace "mak/general.mak", "gcc", ENV.cc
-    system "make"
+    system "make", "CC=#{ENV.cc}", "LD=#{ENV.cc}"
 
     # Makefile doesn't build a dylib
     system ENV.cc, "-dynamiclib", "-install_name", "#{lib}/libmdxmini.dylib",
```
Comment (SMillerDev) [inline Formula/mdxmini.rb:30]: Can we contribute this upstream?

File: Formula/mdxmini.rb [hunk 2]
```diff
@@ -53,7 +51,7 @@ def install
           printf("%s\\n", title);
       }
     EOS
-    system ENV.cc, "mdxtest.c", "-L#{lib}", "-lmdxmini", "-o", "mdxtest"
+    system ENV.cc, "mdxtest.c", "-L#{lib}", "-L#{HOMEBREW_PREFIX}/lib", "-lmdxmini", "-lSDL2", "-o", "mdxtest"
```
Comment (SMillerDev) [inline Formula/mdxmini.rb:54]:
```suggestion
    system ENV.cc, "mdxtest.c", "-L#{lib}", "-L#{Formula["sdl2"].opt_lib}", "-lmdxmini", "-lSDL2", "-o", "mdxtest"
```

File: Formula/mdxmini.rb (hunk 1 above)
Comment (carlocab) [reply Formula/mdxmini.rb:30]: Sent a patch that should make it easier to build a shared library. https://github.com/mistydemeo/mdxmini/pull/3

**Comment by danielnachun** [comment]
I pushed an additional commit to fix the build on Linux.  I will upstream the patch in another PR now that I know it works on all platforms.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**